schedule/containers/extra_tests_textmode_containers.yaml

```diff
@@ -1,27 +1,37 @@
 name:           extra_tests_textmode_containers
 description:    >
-    Maintainer: jalausuch@suse.com, qa-c@suse.de.
-    Extra tests about software in containers module
+  Maintainer: jalausuch@suse.com, qa-c@suse.de.
+  Extra tests about software in containers module
 conditional_schedule:
-    podman_image:
-        DISTRI:
-            'opensuse':
-                - containers/podman_image
-    docker_image:
-        DISTRI:
-            'opensuse':
-                - containers/docker_image
-                - containers/container_diff
+  boot:
+    ARCH:
+      's390x':
+        - installation/bootloader_start
+        - boot/boot_to_desktop
```

Comment:
since boot_to_desktop is scheduled for all conditions, I would remove it from here and put it directly in the schedule.

Reply:
7f7d1aa26 done